docs/en/sql-reference/functions/time-window-functions.md

```diff
@@ -66,8 +66,6 @@ tumbleStart(time_attr, interval [, timezone]);
 - `interval` — Window interval in [Interval](../data-types/special-data-types/interval.md).
 - `timezone` — [Timezone name](../../operations/server-configuration-parameters/settings.md#timezone) (optional).
 
-The parameters above can also be passed to the function as a [tuple](../data-types/tuple.md).
```

Comment:
(This was incorrect, instead `tumbleStart((x, y)) = x` and `tumbleEnd((x, y)) = y`.)